src/trie.c

```diff
@@ -29,6 +29,7 @@ trie_t *new_trie(char current)
 
     t->is_word = 0;
     t->parent = NULL;
+    
```

Comment:
Looks like someone's editor doesn't strip trailing whitespace. Please revert this change so that this PR doesn't touch this file.

Reply:
How do I do this exactly?

Reply:
I did git reset HEAD src/trie.c, is that it?